### PR TITLE
Use large resouce class for CI system tests

### PR DESCRIPTION
**Pull request by alpe**
better machine type for system tests on CircleCI
System tests are flickering: 
* https://app.circleci.com/pipelines/github/CosmWasm/wasmd/3733/workflows/808a7a32-fb30-4fc5-a41f-7175903da16c/jobs/15866
* https://app.circleci.com/pipelines/github/CosmWasm/wasmd/3724/workflows/b1b80b81-5b0d-4f99-b3e1-368370b20712/jobs/15859?invite=true#step-104-202

### CODE REVIEW

**Comment by alpe** [comment]
Still failing

File: .circleci/config.yml [hunk 1]
```diff
@@ -109,6 +113,17 @@ jobs:
       - run:
           name: Build and run system tests
           command: make test-system
+      - run:
+          name: Build and run system tests
+          command: make test-system
```
Comment (pinosu) [inline .circleci/config.yml:118]: This looks duplicate.. probably should be removed

**Comment by alpe** [comment]
I am going to revert changes to have only the resource class here.

**Review by pinosu — APPROVED** [review]
LGTM 👍

**Pull request merged**